docs/index.md

```diff
@@ -208,13 +111,15 @@ The Operator IAM roles will be created per cluster with the [rosa-sts](https://r
 ## Variables for Red Hat Cloud Services Terraform provider
 
 The following variables are used to create a ROSA cluster with the Terraform provider:
-
-- `client_id` (String) - OpenID client identifier.
-- `client_secret` (String, Sensitive) - OpenID client secret.
-- `insecure` (Boolean) - When set to 'true' enables insecure communication with the server. This disables verification of TLS certificates and host names, and it is not recommended for production environments.
-- `password` (String, Sensitive) - User password.
-- `token` (String, Sensitive) - Access or refresh token that is generated from https://console.redhat.com/openshift/token/rosa.
-- `token_url` (String) - OpenID token URL.
-- `trusted_cas` (String) - PEM encoded certificates of authorities that will be trusted. If this is not explicitly specified, then the provider will trust the default certificate authorities trusted by the system.
-- `url` (String) - URL of the API server.
-- `user` (String) - User name.
+  
+| Parameter | Type | Description |
+|----------|------|-------------|   
+|`client_id` | String | OpenID client identifier. |
+|`client_secret` | String, Sensitive | OpenID client secret. |
+|`insecure` | Boolean | When set to 'true' enables insecure communication with the server. This disables verification of TLS certificates and host names, and it is not recommended for production environments. |
+|`password` | String, Sensitive | User password. |
+|`token` | String, Sensitive | Access or refresh token that is generated from https://console.redhat.com/openshift/token/rosa. |
+|`token_url` | String | OpenID token URL. |
+|`trusted_cas` | String | PEM encoded certificates of authorities that will be trusted. If this is not explicitly specified, then the provider will trust the default certificate authorities trusted by the system. |
+|`url` | String | URL of the API server. |
```

Comment:
instead of saying "URL of the API server." why not just "api.openshift.com"  because that's not going to change for end users.

Reply:
We might remove some of the parameters from the documentation as those not expected to be used by external users:
* token_url
* url
* insecure?

About the other attributes, we will have to have specific guide for the accessing credentials, something similar to what AWS created here: https://registry.terraform.io/providers/hashicorp/aws/latest/docs#authentication-and-configuration